<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1292</width>
    <height>882</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RvSimpleEmulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="toolBarWidget" native="true">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>20</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <property name="horizontalSpacing">
      <number>3</number>
     </property>
     <item row="0" column="0">
      <widget class="QToolButton" name="runButton">
       <property name="toolTip">
        <string>Run</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="media-playback-start"/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QToolButton" name="pauseButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Pause</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="media-playback-pause"/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="reloadButton">
       <property name="toolTip">
        <string>Reload</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="media-playlist-repeat"/>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QToolButton" name="stepButton">
       <property name="toolTip">
        <string>Step</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="go-next"/>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QToolButton" name="breakButton">
       <property name="toolTip">
        <string>Toggle Breakpoint</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="media-record"/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="centralWidget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>1041</width>
      <height>651</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0" rowspan="2" colspan="2">
      <widget class="QListWidget" name="instListWidget"/>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QListWidget" name="regListWidget"/>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QCheckBox" name="hexDisplayCheckBox">
       <property name="text">
        <string>Hex</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="4">
      <widget class="QListWidget" name="memListWidget"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Address:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLineEdit" name="memTargetEdit">
       <property name="placeholderText">
        <string>Hex</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="memJumpButton">
       <property name="text">
        <string>Follow</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1292</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionLicenses"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionLicenses">
   <property name="text">
    <string>Licenses</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>645</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLicenses</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>licensePage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>645</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>aboutPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>645</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>645</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MainWindow</sender>
   <signal>sigCpuPause()</signal>
   <receiver>MainWindow</receiver>
   <slot>cpuPause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>645</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>645</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pauseButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>cpuPause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1025</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>645</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>runButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>cpuRun()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>990</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>645</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stepButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>cpuStep()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1095</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>645</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reloadButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>cpuReload()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1060</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>645</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>breakButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>cpuBreak()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1130</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>645</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSettings</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>settingsPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>645</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hexDisplayCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>updateReg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>807</x>
     <y>427</y>
    </hint>
    <hint type="destinationlabel">
     <x>645</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>memJumpButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>memJump()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>934</x>
     <y>744</y>
    </hint>
    <hint type="destinationlabel">
     <x>645</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>sigCpuPause()</signal>
  <slot>openFile()</slot>
  <slot>aboutPage()</slot>
  <slot>licensePage()</slot>
  <slot>cpuRun()</slot>
  <slot>cpuReload()</slot>
  <slot>cpuPause()</slot>
  <slot>cpuStep()</slot>
  <slot>cpuBreak()</slot>
  <slot>settingsPage()</slot>
  <slot>updateReg()</slot>
  <slot>memJump()</slot>
 </slots>
</ui>
